src/App.tsx

```diff
@@ -1,19 +1,70 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { FilterBy } from './types/FilterBy';
+import { TodoList } from './components/TodoList';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.ALL);
+  const [errorStatus, setErrorStatus] = useState(false);
+
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const getTodosFromApi = async () => {
+    if (user) {
+      try {
+        const todosApi = await getTodos(user?.id);
+
+        setTodos(todosApi);
+      } catch {
+        setErrorStatus(true);
+
+        setTimeout(() => {
+          setErrorStatus(false);
+        }, 3000);
+      }
+    }
+  };
+
   useEffect(() => {
+    getTodosFromApi();
     // focus the element with `ref={newTodoField}`
     if (newTodoField.current) {
       newTodoField.current.focus();
     }
   }, []);
 
+  const visibleTodos = todos.filter(todo => {
+    switch (filterBy) {
+      case FilterBy.ACTIVE:
+        return !todo.completed;
+      case FilterBy.COMPLETED:
+        return todo.completed;
+      default:
+        return todo;
+    }
+  });
```

Comment:
It's better to keep visibleTodos in the state then a separate variable in the component